osidb/models.py

```diff
@@ -1349,20 +1349,34 @@ def _perform_bzsync(self, *args, bz_api_key=None, no_alerts=False, **kwargs):
 
         creating = self.bz_id is None
 
-        # sync to Bugzilla
-        bs = FlawBugzillaSaver(self, bz_api_key)  # prepare data for save to BZ
-        flaw_instance = bs.save()  # actually send to BZ (but not save to DB)
-
-        if creating:
-            # Save bz_id to DB
-            kwargs["auto_timestamps"] = False  # no timestamps changes on save to BZ
-            kwargs["raise_validation_error"] = False  # the validations were already run
-            # save in case a new Bugzilla ID was obtained
-            # Instead of self.save(*args, **kwargs), just update the single field to avoid
-            # race conditions.
-            flaw_instance.save(
-                *args, update_fields=["meta_attr"], no_alerts=no_alerts, **kwargs
+        try:
+            # sync to Bugzilla
+            bs = FlawBugzillaSaver(self, bz_api_key)  # prepare data for save to BZ
+            flaw_instance = bs.save()  # actually send to BZ (but not save to DB)
+
+            if creating:
+                # Save bz_id to DB
+                kwargs["auto_timestamps"] = False  # no timestamps changes on save to BZ
+                kwargs[
+                    "raise_validation_error"
+                ] = False  # the validations were already run
+                # save in case a new Bugzilla ID was obtained
+                # Instead of self.save(*args, **kwargs), just update the single field to avoid
+                # race conditions.
+                flaw_instance.save(
+                    *args, update_fields=["meta_attr"], no_alerts=no_alerts, **kwargs
+                )
+        except Exception as e:
+            # Sync failed but if it was done async the original flaw may be saved, resulting in
+            # incosnsitent data between OSIDB and BZ.
+            logger.error(f"Error when syncing flaw {self.uuid} to Bugzilla: {e}.")
+            self.alert(
+                "bzsync_failed",
+                "The Bugzilla sync for this flaw failed in the last save, so there may be data discrepancies. "
+                "Our team has been notified about this and is looking into it.",
```

Comment:
`Our team` does not sound very specific. I would either replace it with `Vulnerability Tooling team` or somehow mention the OSIDB developers instead